src/core/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Menu } from '@material-ui/core';
+import { useStyles } from './styles';
+import { ButtonFilled } from '../Button/ButtonFilled';
+
+import { BasePropsMenu } from './base';
+
+interface CustomDropdownProps extends BasePropsMenu {
+  anchorEl: HTMLElement | null;
+}
+const Dropdown: React.FC<CustomDropdownProps> = ({ children, anchorEl }) => {
+  const classes = useStyles();
+
+  const [anchorElement, setAnchorElement] = React.useState<null | HTMLElement>(
+    anchorEl
+  );
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorElement(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorElement(null);
+  };
+  const showDropdown = Boolean(anchorElement) ? classes.show : classes.hide;
+  return (
+    <div className={classes.root} data-id="dropdownElement">
+      <ButtonFilled
```

Comment:
button to open the menu shouldn't be part of the component...because we have the component in header and all where there is no button

Reply:
Do I have to add button in stories and pass click event to component? or how else can I do it?